### ISSUE TITLE
Can't import compass_twitter_bootstrap_responsive

### ISSUE THREAD

**Issue by chrisnicola** [vwall/compass-twitter-bootstrap#19]
Get an error with the imports:

```
File to import not found or unreadable: variables.
Load paths:
  Sass::Rails::Importer(/Users/chrisn/dev/pixelpublish/app/assets/stylesheets/application.css.scss)
  /Users/chrisn/.rvm/gems/ruby-1.9.2-p290/gems/compass-0.11.7/frameworks/compass/stylesheets
  /Users/chrisn/.rvm/gems/ruby-1.9.2-p290/bundler/gems/compass-twitter-bootstrap-d6c049504fcb/stylesheets
  (in /Users/chrisn/dev/pixelpublish/app/assets/stylesheets/application.css.scss)
```

**Comment by vwall** [comment]
What does application.css.scss look like?

**Comment by vwall** [comment]
Should be fixed with 3e2434603edcbb7559c92c83565bac6ca2500890
